src/main/java/com/intel/genomicsdb/model/CommandLineImportConfig.java

```diff
@@ -48,6 +64,8 @@ public CommandLineImportConfig(final String command, final String[] commandArgs)
         longopts[6] = new LongOpt("callset-output", LongOpt.REQUIRED_ARGUMENT, null, ArgsIdxEnum.ARGS_IDX_CALLSET_OUTPUT.idx());
         longopts[7] = new LongOpt("pass-as-bcf", LongOpt.NO_ARGUMENT, null, ArgsIdxEnum.ARGS_IDX_PASS_AS_BCF.idx());
         longopts[8] = new LongOpt("vcf-header-output", LongOpt.REQUIRED_ARGUMENT, null, ArgsIdxEnum.ARGS_IDX_VCF_HEADER_OUTPUT.idx());
+        longopts[9] = new LongOpt("size_per_column_partition", LongOpt.REQUIRED_ARGUMENT, null, ArgsIdxEnum.ARGS_IDX_SIZE_PER_COLUMN_PARTITION.idx());
+        longopts[10] = new LongOpt("segment_size", LongOpt.REQUIRED_ARGUMENT, null, ArgsIdxEnum.ARGS_IDX_SEGMENT_SIZE.idx());
```

Comment:
Protobuf spec says:

message ImportConfiguration {
  required int64 size_per_column_partition = 7 [default = 16384];
...

changing it to:
message ImportConfiguration {
  optional int64 size_per_column_partition = 7;
...

Please, confirm.

Reply:
My bad, please ignore my previous comment.
* ```size_per_column_partition``` should remain required in Protobuf
* Please call ```configurationBuilder.setSizePerColumnPartition(<DEFAULT_VALUE>)```  here in case the user doesn't provide a value in the command line for this parameter